ansible_collections/arista/avd/examples/isis-ldp-ipvpn/README.md

```diff
@@ -302,7 +298,7 @@ p2p_uplinks_mtu: 1500 # (4)!
 ```
 
 1. The name of the fabric for internal AVD use. This name *must* match the name of an Ansible Group (and therefore a corresponding group_vars file) covering all network devices.
-2. Local users/passwords and their privilege levels. In this case, the `ansible` user is set with the password `ansible`, and an `admin` user is set with no password.
+2. Local users/passwords and their privilege levels. In this case, the `ansible` user is set with the password `ansible`.
```

Comment:
For the example it would be better to just rename ansible-> arista and keep the admin as without password.